src/openrct2-ui/UiContext.Linux.cpp

```diff
@@ -23,6 +23,7 @@
 #include <openrct2/core/String.hpp>
 #include <openrct2/ui/UiContext.h>
 #include "UiContext.h"
+#include "openrct2/localisation/localisation.h"
```

Comment:
This is using `libopenrct2`, it should use angled brackets and it should respect the sort order

Reply:
I tried using angled brackets, it wouldn't compile... And can you clarify sort order?